test/base/driver_util.py

```diff
@@ -851,6 +856,8 @@ def setup(self, config_object=None):
                     datatypes_conf=datatypes_conf_override,
                     prefer_template_database=getattr(config_object, "prefer_template_database", False),
                     log_format=log_format,
+                    conda_auto_init=self.conda_auto_init,
+                    conda_auto_install=self.conda_auto_install,
```

Comment:
Should these have value `getattr(config_object, 'conda_auto_init', False)` and `getattr(config_object, 'conda_auto_install', False)` respectively?
If yes, then probably the new lines 799-800 are not needed.

Reply:
Yes, that should work!